### PR TITLE
subnational pop basemap files

### PR DESCRIPTION
**Pull request by jordan-klein**

### CODE REVIEW

File: scripts/data_processing/PPI/basemap_mvp.R [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+    st_set_geometry(.$geometry)
+
+### Clear memory
+rm(shapefile, shapefile_raw, shapes_chn_twn)
+gc()
+
+#### 3) Get get level 1 & 2 subnational divisions for each country ####
+##### AUS- only 1st level
+
+##### BEL- have 3rd level, aggregate up to 2nd (treat as 1st)
```
Comment (kaitejohnson) [inline scripts/data_processing/PPI/basemap_mvp.R:67]: Is this what you mean by a canonical admin level 1 -- figuring out for each country what the admin level 1 should be ? Is this standardized somewhere else?

File: scripts/data_processing/PPI/basemap_mvp.R [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+##### SAU- only 1st level
+
+##### SWE- only 1st level
+
+##### THA- pop at level 2, level 1 in the data
+
+##### TWN- only 1st level
+
+##### USA- pop at level 2, level 1 in the data
```
Comment (kaitejohnson) [inline scripts/data_processing/PPI/basemap_mvp.R:224]: Question, are these placeholders to do the aggregation?? I.e. in the Beckett data we have population by country, and want to aggregate to state as well?  What do you mean when you say level 1 in the data?

File: scripts/data_processing/PPI/subnational_population_basemap_relational_README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Description of scripts used to create the relational data files for the 26-country subnational population basemap by Jordan Klein
+
+### basemap_mvp.R
```
Comment (kaitejohnson) [inline scripts/data_processing/PPI/subnational_population_basemap_relational_README.md:3]: This makes a lot more sense to me now! Thanks for aggregating in this way!

**Review by kaitejohnson — COMMENTED** [review]
Hi Jordan,

This looks great! Just left a few remaining questions/comments more out of curiosity than anything!

File: scripts/data_processing/PPI/basemap_mvp.R (hunk 1 above)
Comment (jordan-klein) [reply scripts/data_processing/PPI/basemap_mvp.R:67]: These are just notes for what I have in the data.

File: scripts/data_processing/PPI/basemap_mvp.R [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+ADM1 <- filter(Geo.Pop, iso3 == "BRA") %>% 
+    select(iso3, cntry_n, adm1_id = micr_nm, pop) %>% 
+    bind_rows(ADM1, .) %>% 
+    add_column(adm1_name = str_to_title(.$adm1_id), .after = "adm1_id")
+
+# Check to make sure nrows of adm1 keys in data = rows in adm1 table
+select(Geo.Pop_subdivs, iso3, adm1_id) %>% st_drop_geometry %>% 
+    unique %>% nrow == nrow(ADM1)
+
+#### 6) Get ADM2 table ####
```
[the remark this answers is not in the document]
Comment (jordan-klein) [reply scripts/data_processing/PPI/basemap_mvp.R:275]: Yes

**Pull request merged**